Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -110,7 +111,8 @@ void DSPCallback(u64 userdata, s64 cyclesLate)
 int GetAudioDMACallbackPeriod()
 {
   // System internal sample rate is fixed at 32KHz * 4 (16bit Stereo) / 32 bytes DMA
-  return s_cpu_core_clock / (AudioInterface::GetAIDSampleRate() * 4 / 32);
+  return static_cast<u64>(s_cpu_core_clock) * AudioInterface::GetAIDSampleRateDivisor() /
+         (Mixer::FIXED_SAMPLE_RATE_DIVIDEND * 4 / 32);
```

Comment:
Minor performance optimization: You should be able to avoid the division by 32 here by instead multiplying with 32 on the left-hand side of the equation.

Reply:
Mixer::FIXED_SAMPLE_RATE_DIVIDEND is a constant expression, so shouldn't the compiler optimize that / 32 out?

Reply:
That's true. You can skip my suggestion then.